jazz_test-lambda/index.js

```diff
@@ -0,0 +1,108 @@
+// =========================================================================
+// Copyright � 2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+/**
+Nodejs Template Project
+@author:
+@version: 1.0
+ **/
+'use strict';
+const errorHandlerModule = require("./components/error-handler.js"); //Import the error codes module.
+const responseObj = require("./components/response.js"); //Import the response module.
+const configObj = require("./components/config.js"); //Import the environment data.
+const logger = require("./components/logger.js"); //Import the logging module.
+const validateARN = require("./components/validate-arn.js");
+const validateJSON = require("./components/validate-json.js");
+var handler = (event, context, cb) => {
+
+  //Initializations
+  var errorHandler = errorHandlerModule();
+  var config = configObj(event);
+  logger.init(event, context);
+  var AWS_REGION =  config.AWS_REGION;
+  try {
+    var testResponse = {
+      "StatusCode": 200,
+      "execStatus": 0
+    }
+    var requestJson
+    if (event !== undefined && event.method !== undefined && event.method === 'POST') {
+      if (!event.body) {
+        return cb(JSON.stringify(errorHandler.throwInputValidationError("Event Body not Defined")));
+      } else if (!event.body.functionARN) {
+        return cb(JSON.stringify(errorHandler.throwInputValidationError("No function ARN provided")));
+      } else if (!validateARN(event.body.functionARN)) {
+        return cb(JSON.stringify(errorHandler.throwInputValidationError("Function ARN is invalid")));
+      } else if (!event.body.inputJSON) {
+        return cb(JSON.stringify(errorHandler.throwInputValidationError("Input for function is not defined")));
+      } else {
+        var functionARN = event.body.functionARN;
+        
+        if (event.body.inputJSON && !validateJSON(event.body.inputJSON)) {
+          return cb(JSON.stringify(errorHandler.throwInputValidationError("Input for function is an invalid JSON")));
+        }
+        if(event.body.region && event.body.region != "" ){
+          AWS_REGION =  event.body.region // If Request Specifies AWS_REGION || Over rides the Configuration value 
+        }
+        var inputJSON = JSON.parse(event.body.inputJSON);
+        invokeLambda(functionARN,inputJSON).then((data) => {

```

Comment:
AWS_REGION is not being passed to invokeLambda and neither is the value being used to construct the inputJSON. How does this thing work?